src/Math/include/BipedalLocomotion/Math/Wrench.h

```diff
@@ -26,22 +26,10 @@ template <class Scalar> class Wrench : public Eigen::Matrix<Scalar, 6, 1>
 public:
     using Base = Eigen::Matrix<Scalar, 6, 1>;
 
-    Wrench()
-        : Eigen::Matrix<Scalar, 6, 1>()
-    {
-    }
-
-    template <typename OtherDerived>
-    Wrench(const Eigen::MatrixBase<OtherDerived>& other)
-        : Eigen::Matrix<Scalar, 6, 1>(other)
-    {
-    }
-
-    template <typename OtherDerived> Wrench& operator=(const Eigen::MatrixBase<OtherDerived>& other)
-    {
-        this->Base::operator=(other);
-        return *this;
-    }
+    /**
+     * Inherit Eigen constructors
+     */
+    using Base::Base;
```

Comment:
Is this actually needed? It is a good practice to avoid defining custom constructors/assign operators if not needed, but I am not sure it is necessary to specify that we are using the base constructors of a publicly inherited class :thinking:

Reply:
The constructor that we need is 
```cpp
Wrench(Eigen::Index rows, Eigen::Index cols)
    : Eigen::Matrix<Scalar, 6, 1>(rows, cols)
{
}
```

I added all of them to avoid problems in the future but if you want I can add only this

Reply:
Sorry my bad. I did not remember that base constructors are not inherited. I think it is ok as it is.